frictionless/detector.py

```diff
@@ -122,48 +124,181 @@ def __setattr__(self, name, value):
 
     @property
     def buffer_size(self):
+        """Returns buffer size of the detector. Default value is 10000.
+        Returns:
+            int: detector buffer size
+        """
         return self.__buffer_size
 
+    @buffer_size.setter
+    def buffer_size(self, value: int):
+        """Sets buffer size for detector.
+        Args:
```

Comment:
We need to use "Parameters:" instead of "Args:" to make `make docs` work as far as I know